docs/contribute.rst

```diff
@@ -232,3 +232,42 @@ make tags
 
 make clean
     Cleanup all temporary files.
+
+
+Release
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Follow the steps below to create a new major or minor release:
+
+* Run the full test coverage using ``tmt -c how=full run``
+* Use ``git log --oneline x.y-1..`` to generate changelog
+* Add a ``Release tmt-x.y.0`` commit with specfile update
+* Create a pull request with the commit, ensure tests pass
+* Tag the commit with ``x.y.0``, push tags ``git push --tags``
+* Create a new `github release`__ based on the tag above
+
+Release a new package to Fedora and EPEL repositories:
+
+* Move the ``fedora`` branch to point to the new release
+* Create a source rpm using the ``make srpm`` command
+* Enable Fedora kerberos ``kinit nick@FEDORAPROJECT.ORG``
+* Change to the fedora rpms git or ``fedpkg clone tmt``
+* Check out the rawhide branch ``git checkout rawhide``
+* Import the srpm using ``fedpkg import /path/to/the/srpm``
+* Restore any files removed by fedpkg if necessary
+* Ensure the proposed changes are ok and commit them
+* Create a pull request against rawhide from your fork
+* After tests pass, merge the pull request to rawhide
+* Build the package for rawhide ``fedkpkg build --nowait``
+* Build package for all `active releases`__
+  ``git checkout f33 && git merge rawhide && git push && fedpkg build --nowait``
+* Create a bodhi update for each release
+  ``git checkout f33 && fedpkg update --type enhancement --notes 'Update title'``
```

Comment:
What do you usually put into `--notes`? I guess that `Update title` is just a placeholder, right?

Reply:
Yeah, placeholder. Usually I just copy & paste the first line from the [release](https://github.com/psss/tmt/releases/tag/1.7.0) notes.